mars/dataframe/core.py

```diff
@@ -1021,6 +1021,13 @@ def index(self):
     def axes(self):
         return [self.index]
 
+    @property
+    def empty(self):
+        shape = getattr(self, '_shape')
+        if np.any(np.isnan(shape)):
+            raise ValueError('Tileable object must be executed first')
```

Comment:
Please add a unittest for Series as well.

Reply:
By `unittest`, you mean something like `a[a>0]` where `a` is a randomly initialized Series, like what I did for dataframe to make it raise ValueError?

Reply:
Right, the code I mentioned is not covered by any tests.

Reply:
Hi @qinxuye, I have rebased to the latest commit and also added the unitest for Series data. Plz have a look ~

Reply:
Thanks for you contribution, so far it looks good, I will review it after CI finished.